test/utility/property_tree.cpp

```diff
@@ -45,8 +47,9 @@ BOOST_AUTO_TEST_CASE(property_tree__property_tree__value__expected_empty)
     {
         "{\"sequence\":\"101\" \"arguments\";\"test\"}"
     };
```

Comment:
Scopes are okay (in terms of wrapping style).

Reply:
:+1:

Updated